recipes-bsp/firmware/firmware-qcom-boot-qcs9100_00006.0.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "QCOM NHLOS Firmware for Qualcomm QCS9100 platform"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://${UNPACKDIR}/${BOOTBINARIES}/Qualcomm-Technologies-Inc.-Proprietary;md5=58d50a3d36f27f1a1e6089308a49b403"
+
+FW_ARTIFACTORY = "softwarecenter.qualcomm.com/download/software/chip/qualcomm_linux-spf-1-0/qualcomm-linux-spf-1-0_test_device_integrationandtest_publictest"
+FW_BUILD_ID = "r1.0_${PV}/qcs9100-le-1-0"
+FW_BIN_PATH = "common/build/ufs/bin"
+BOOTBINARIES = "QCS9100_bootbinaries"
+
+SRC_URI = "https://${FW_ARTIFACTORY}/${FW_BUILD_ID}/${FW_BIN_PATH}/${BOOTBINARIES}.zip"
+SRC_URI[sha256sum] = "480682759e27d63b0e44501ae2517b3671bea6dad21071880a22ed5feb5a458b"
+
+S = "${WORKDIR}/sources"
+UNPACKDIR = "${S}"
+
+INHIBIT_DEFAULT_DEPS = "1"
+
+do_configure[noexec] = "1"
+do_compile[noexec] = "1"
```

Comment:
I think few previous strings should go to `firmware-qcom-boot-common.inc`, they are repetitive between recipes.

Reply:
Yeah, @adhudase try to move common pieces to firmware-qcom-boot-common.inc.

Reply:
Done, Thanks.